roadmap/self-hosting/README.md

```diff
@@ -42,6 +40,10 @@ Post-install, configuration options can be modified by the administrator within
 - The URL of the main Sandstorm UI. (`BASE_URL`) Changing this also resets the OAuth configuration, since OAuth identity providers require that you specify the callback URL when creating the client key.
 - The IP address and port to which the server binds. (`BIND_IP` and `PORT`)
 
+### TODO(feature) Docker
```

Comment:
Two notes here:
1. I can't merge requests that amend the roadmap in addition to the docs.
2. This roadmap document probably shouldn't have items added to it via community contributors. It is more or less as Kenton wrote/envisioned it. We use issues/feature requests to track things we'd like to happen on the project.